src/app/components/Sidebar/Sidebar.jsx

```diff
@@ -15,7 +17,8 @@ const Sidebar = (props) => {
   const updateCurrentListSelectorValues = (data) => {
     // Deep clones the object of props.listOptions to a new object so the update later will not
     // affect the original props.listOptions
-    const listOptions = JSON.parse(JSON.stringify(props.listOptions));
+    const listOptions = (props.listOptions && !_isEmpty(props.listOptions)) ?
```

Comment:
You passed `props` as an argument on line 68, so it's the same as `data`. Use one.